test/osiris_SUITE.erl

```diff
@@ -468,6 +469,17 @@ single_node_offset_listener(Config) ->
     flush(),
     ok.
 
+osiris_reader_set_current_file(Config) ->
+    Name = ?config(cluster_name, Config),
+    Conf0 =
+        #{name => Name,
+          epoch => 1,
+          leader_node => node(),
+          replica_nodes => []},
+    {ok, #{leader_pid := Leader}} = osiris:start_cluster(Conf0),
+    {ok, Log0} = osiris:init_reader(Leader, next, {offset_reader_1, []}),
+    ?assertNot(undefined =:= osiris_log:get_current_file(Log0)).
```

Comment:
Looks like the dialyzer isn't happy about this check since `get_current_file/1`'s spec says that it can never return `undefined`. Since we're opening this on an empty log we could check this instead:


```suggestion
    ?assertEqual(<<"00000000000000000000.segment">>, 
                 osiris_log:get_current_file(Log0)).
```